### PR TITLE
Add doc to generate indexed rasters

### PR DESCRIPTION
**Pull request by ger-benjamin**
Add Missing/Misplaced steps...

### CODE REVIEW

File: doc/administrator/mapfile.rst [hunk 1]
```diff
@@ -406,6 +406,29 @@ Performance improvement
 Adding an ``EXTENT`` parameter to the ``LAYER`` section may significantly improve the performances
 because it saves MapServer from computing the extent of all layer features.
 
+Generating shapefile as a raster tileindex
```
Comment (sbrunner) [inline doc/administrator/mapfile.rst]: Prepare raster files

File: doc/administrator/mapfile.rst [hunk 2]
```diff
@@ -406,6 +406,29 @@ Performance improvement
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you need raster tileindex (e.g. from a .TIF field picture) you can generate a
+corresponding shapfile by doing these steps:
```
Comment (sbrunner) [inline doc/administrator/mapfile.rst]: To have good performance you should have tiled files with overview, and probably a tileindex, you can ...

File: doc/administrator/mapfile.rst [hunk 3]
```diff
@@ -406,6 +406,29 @@ Performance improvement
...
+
+Convert your rasters in tiled GTIF:
```
Comment (sbrunner) [inline doc/administrator/mapfile.rst]: GTIF => GeoTIFF

**Comment by sbrunner** [comment]
Just small comment, be free to adapt them :-)

**Comment by ger-benjamin** [comment]
Thanks @sbrunner

**Comment by sbrunner** [comment]
Thanks, merge :-)

**Pull request merged**